index.js

```diff
@@ -49,6 +49,7 @@ const noop = () => {};
  * @type {Object}
  * @property {BlobPropertyBag} [blobOptions]
  * @property {Boolean} [recordScreen]
+ * @property {MediaStream} customMediaStream
```

Comment:
Small fix we could wrap this in square brackets `[customMediaStream]` to indicate that it's an optional parameter